packages/api/test/scripts/test.env

```diff
@@ -0,0 +1,30 @@
+ENV=test
+DEBUG=true
+SALT=secret
+DATABASE_URL=http://127.0.0.1:3000
+DATABASE_TOKEN=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJzdXBhYmFzZSIsImlhdCI6MTYwMzk2ODgzNCwiZXhwIjoyNTUwNjUzNjM0LCJyb2xlIjoic2VydmljZV9yb2xlIn0.necIJaiP7X2T2QjGeV-FhpkizcNTX8HjDDBAxpgQTEI
+
+MAGIC_SECRET_KEY=test
+MAILCHIMP_API_KEY=
+METAPLEX_AUTH_TOKEN=metaplex-test-token
+LOGTAIL_TOKEN=
+PRIVATE_KEY=xmbtWjE9eYuAxae9G65lQSkw36HV6H+0LSFq2aKqVwY=
+SENTRY_DSN=https://test@test.ingest.sentry.io/0000000
+SLACK_USER_REQUEST_WEBHOOK_URL=test
+
+CLUSTER_API_URL=http://127.0.0.1:9094
+# will be used with we can active auth in cluster base64 of test:test
+CLUSTER_BASIC_AUTH_TOKEN=dGVzdDp0ZXN0
+CLUSTER_SERVICE=
+
+MAINTENANCE_MODE=rw
+
+S3_ENDPOINT=http://127.0.0.1:9000
+S3_REGION=us-east-1
+S3_ACCESS_KEY_ID=minioadmin
+S3_SECRET_ACCESS_KEY=minioadmin
+S3_BUCKET_NAME=dotstorage-test-0
+
+NFT_STORAGE_VERSION=test-version
+NFT_STORAGE_BRANCH=test-branch
+NFT_STORAGE_COMMITHASH=test-commit
```

Comment:
For now we can keep this. We should create an issue to track having a single place for all these

Reply:
Hmm... actually, we might not need this after all.

I added this early in the process hoping we'd be able to get rid of it, and now that I've thought a bit more, the only differences between this and the dev `env.tpl` are the service ports, which are overriden anyway, and the S3 bucket name. But since dev and test will be using different minio images, so they can have the same bucket name without collisions, so we can just use `dotstorage-dev-0` for both.

Let me see if I can get things passing without this file - that would be sweet.